src/styles/blocks/cal.scss

```diff
@@ -0,0 +1,46 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * $cell-size;
+  border: 1px solid #000;
+
+  $days:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
+
+  @each $day, $date in $days {
+    &--start-day--#{$day}:first-child {
+      margin-left: $cell-size * $date + $cell-gap * $date;
```

Comment:
use parenthesis for accurate calculation

Reply:
fixed ;)